CTCPreferencesController.m

```diff
@@ -46,6 +50,8 @@ - (void)refreshInvalidInputMarkers {
     NSImage *invalidImage = [NSImage imageNamed:@"warning"];
     self.torrentsSavePathWarningImageView.image = CTCDefaults.isTorrentsSavePathValid ? validImage : invalidImage;
     self.feedURLWarningImageView.image = CTCDefaults.isFeedURLValid ? validImage : invalidImage;
+    
+    [CTCScheduler.sharedScheduler updateAppNapStatus];
```

Comment:
The point of this method is to update the UI (namely, the warning markers), so it shouldn't be calling into code that changes the power assertions.